unicoremc/views.py

```diff
@@ -35,7 +38,13 @@ def get_all_repos(request):
             break
         repos.extend(data)
         pageNum += 1
+    cache.set('repos', repos)
```

Comment:
no, it's indefinite. That call is very slow, so refreshing the repo cache should be manual.